### DISCUSSION TITLE
Your website is linking to a malicious webpage

### DISCUSSION THREAD

**Discussion by rpbp58** [xtekky/gpt4free#357] [General]
When I click https://chat.g4f.ai/chat it redirects to https://www.profitabledisplaynetwork.com

What's going on?

**Comment by JMcrafter26** [comment]
This site: https://www.profitabledisplaynetwork.com/ is an ad server, which helps them to keep the project alive. 
If you cannot visit the site at all because of this, just turn your adblocker on.
For me, the site works fine with and without the adblocker on.

JMcrafter26
-Happy coding 🎈
